geode-core/build.gradle

```diff
@@ -233,6 +233,8 @@ dependencies {
         exclude group: '*'
     }
 
+    runtimeOnly(project(':geode-deployment-legacy'))
```

Comment:
After looking at the contents of `geode-core/build.gradle` I am not sure what this line is buying us on the classpath or POM file.

Reply:
I wasn't considering the actual class files from the new service.

Reply:
The deployment code has been decoupled and replaced with a JarDeploymentService interface. The runtime dependency is it ensure that we have a runtime dependency on this module, but not compile time.